eosclubhouse/clubhouse.py

```diff
@@ -62,6 +62,7 @@
 CLUBHOUSE_NAME = 'com.hack_computer.Clubhouse'
 CLUBHOUSE_PATH = '/com/hack_computer/Clubhouse'
 CLUBHOUSE_IFACE = CLUBHOUSE_NAME
+USE_INAPP_NOTIFICATIONS = False
```

Comment:
Ah, just saw this. So is not detecting Wayland. Is using the extension always. :+1: then, good to go!